frontend/src/views/critical-ratio.js

```diff
@@ -0,0 +1,112 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import plane from "../images/plane.png";
+import { useRound } from "../selectors";
+import { useAddCriticalRatio } from "../action-hooks";
+import { Button } from "../components/ui";
+
+function CriticalRatio() {
+  const [criticalRatio, setCriticalRatio] = React.useState("");
+  const [isEnabled, setIsEnabled] = React.useState(false);
+
+  const addCriticalRatio = useAddCriticalRatio();
+
+  const state = useRound();
+
+  const buttoncomponent = {
+    width: 170,
+    display: "block",
+    backgroundColor: "black",
+    color: "white",
+    padding: "14px 20px",
+    margin: "10px auto",
+    border: "none",
+    borderRadius: 4,
+    cursor: "pointer",
+    fontSize: "15px",
+    letterSpacing: ".25px",
+    justifyContent: "center"
+  };
+
+  const styleInput = {
+    margin: "10px auto",
+    background: "none",
+    border: "1px solid black",
+    borderRadius: ".25rem",
+    boxSizing: "border-box",
+    display: "block",
+    fontSize: "1rem",
+    padding: ".5rem",
+    color: "#34495e",
+    width: "100%",
+    "&:focus": {
+      outline: "none",
+      borderColor: "rgba(242, 107, 117, 0.5)"
+    }
+  };
+
+  const contenedor = {
+    flexDirection: "column",
+    display: "flex",
+    justifyContent: "center",
+    maxWidth: "400px",
+    margin: "auto",
+    marginTop: 50
+  };
+  const fieldset = { border: "none", margin: "20px", padding: "0" };
+
+  const labelcss = { display: "flex", justifyContent: "center", marginTop: 10 };
+  const databox = {
+    display: "flex",
+    justifyContent: "center",
+    marginTop: 10,
+    background: "#EFF3F1"
+  };
+  const imagecss = {
+    display: "flex",
+    justifyContent: "center",
+    margin: "auto"
+  };
+
+  function onChangeInputCriticalRatio(event) {
+    setCriticalRatio(event.target.value);
+    setIsEnabled(criticalRatio.length > 0);
+  }
+  function handleSubmit(event) {
+    event.preventDefault();
+    addCriticalRatio(criticalRatio);
+  }
+  return (
+    <form css={contenedor} onSubmit={handleSubmit}>
+      <img css={imagecss} src={plane} alt="plane" />
+      <div css={labelcss}>Flight PDR456</div>
+      <div css={databox}>
+        <div css={fieldset}>
```

Comment:
This element contains relevant text. What do you think about using others tags more semantic like <p>, <span>, <b>, etc. 

Maybe: 
```
        <p css={fieldset}>
          Underage Cost<b>{state.underageCost}</b>
        </p>
```